### ISSUE TITLE
 Review Design Wireframe: Support us page

### ISSUE THREAD

**Issue by lem31** [NoroffFEU/musikkforandrerliv.no#91]
- [x] Visual Hierarchy
- [x] Layout
- [x] Consistency
- [x] Screen size
- [x] Usability Principles
- [x] Web Content Accessibility Guidelines (WCAG)
- [x] Colors
- [x] Fonts
- [x] Whitespace and padding
- [x] Image quality
- [x] Iconography
- [x] Micro interactions
- [x] Error states and feedback
- [x] Performance Optimization
- [x] Design documentation

See Design wireframe/prototype checklist for more info

**Comment by Kittypoda** [comment]
I attempted to view the grid on this page in Figma, but I couldn’t find any. It seems like the design team may not have followed a grid for this page. It’s challenging to review the layout and spacing without a grid as a guide.

In the desktop wireframe, the "VOLUNTEER", "DONATION", and "FUNDRAISING" elements have more white space at the bottom than at the top. Perhaps this section would look neater if the spacing were consistent.

In the mobile wireframe, the "VOLUNTEER", "DONATION", and "FUNDRAISING" elements have too much white space, which detracts from the user experience.

The heading "Support" lacks consistency in padding. In some of the mobile wireframes, the heading overlaps with the red section.

In the newsletter section on desktop, the buttons are placed inside the text field, which is a bit confusing and inconsistent with the mobile design. It also lacks some of the text from the mobile design.

In the contact form element, there are links to the Terms & Conditions and Privacy Policy. These links have a contrast ratio of 22.7:1, which does not meet WCAG’s expectations.

**Comment by Kittypoda** [comment]
After my last comment I see that the design team have added a grid on the page, but it dosen't look like they are following it.
<img width="898" alt="Skjermbilde 2024-04-04 kl  15 43 22" src="https://github.com/NoroffFEU/musikkforandrerliv.no/assets/146728692/c109562c-81a0-4226-98fb-1167044506a4">

**Comment by Kittypoda** [comment]
The typo does not follow the style guide where it says that Montserrat should be H1

<img width="1115" alt="Skjermbilde 2024-04-04 kl  15 40 11" src="https://github.com/NoroffFEU/musikkforandrerliv.no/assets/146728692/a9505aa5-6f6f-48ff-b13d-db3e8893f2fd">
<img width="764" alt="Skjermbilde 2024-04-04 kl  15 49 56" src="https://github.com/NoroffFEU/musikkforandrerliv.no/assets/146728692/fa5fd380-9529-4a13-8cfa-6244e45cf6d3">

**Comment by Kittypoda** [comment]
Sections like this do not meet the WCAG criteria for contrast.

<img width="987" alt="Skjermbilde 2024-04-05 kl  09 47 00" src="https://github.com/NoroffFEU/musikkforandrerliv.no/assets/146728692/7f692dfb-6bd4-40b0-b338-21c96e910cb9">

<img width="734" alt="Skjermbilde 2024-04-05 kl  09 19 35" src="https://github.com/NoroffFEU/musikkforandrerliv.no/assets/146728692/45d6b303-87a2-4ba8-8612-8ad4b802b17a">

**Comment by Vickabu** [comment]
Fixed the spacing for section with  "VOLUNTEER", "DONATION", and "FUNDRAISING" on desktop and mobile, and the heding "Support"


"The typography on the "Support Us" page does not adhere to the style guide. According to the style guide, H1 headings should use the Montserrat font, but the font currently used is Just Another Hand." - Just another hand is listed as alternativ font as shown in the snip - however, we have updated the Style Guide to be more clear. 

Red sections like "Thanks for support", "our work" etc is now changed.

"In the contact form element, there are links to the Terms & Conditions and Privacy Policy. These links have a contrast ratio of 22.7:1, which does not meet WCAG’s expectations." - fixed

We are still working on grid, and newsletter components.
